modules/web/src/app/project/edit-project/template.html

```diff
@@ -53,11 +53,8 @@
             <i class="km-os-image-ubuntu"></i>
             Ubuntu
           </mat-button-toggle>
-          <mat-button-toggle [value]="OperatingSystem.CentOS">
-            <i class="km-os-image-centos"></i>
-            CentOS
-          </mat-button-toggle>
-          <mat-button-toggle [value]="OperatingSystem.Flatcar">
+          <mat-button-toggle *ngIf="adminAllowedOperatingSystems.flatcar"
```

Comment:
This line change should be reverted.

Reply:
Same comment for `modules/web/src/app/shared/components/add-project-dialog/template.html`.
These changes don't exist in 2.26